CenterForOpenScience/ember-osf-web
# [Hotfix] Redundant wording for branded registries title

- Ticket: [ENG-2288]
- Feature flag: n/a

## Purpose
Branded registries pages have redundant wording on the tab title

## Summary of Changes
- updated `title` in branded pages template
- remove unused translation in translation file

## Side Effects
`NA`

## QA Notes
- All branded pages should now have the extra `Registries` removed from the tab's title. Please check the branded Discover and branded Add New pages
